PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -3054,10 +3054,9 @@ struct HfTrackIndexSkimCreatorLfCascades {
   }
 
   Filter filterSelectCollisions = (aod::hf_sel_collision::whyRejectColl == 0);
-  Filter filterSelectTrackIds = (aod::hf_sel_track::isSelProng >= 4); // select tracks passing bachelor selection
```

Comment:
Here I would keep the filter, to at least remove already all the tracks that are not selected by any cut as done for 2 and 3 prongs https://github.com/AliceO2Group/O2Physics/blob/c55ebbfbe855c4774fce3e9ac5a5978ab97f4a13/PWGHF/TableProducer/trackIndexSkimCreator.cxx#L1094.